db/migrate/20240605125438_seed_autogrant_limits.rb

```diff
@@ -0,0 +1,5 @@
+class SeedAutograntLimits < ActiveRecord::Migration[7.1]
+  def up
+    load Rails.root.join("db/seeds/autogrant_limits.rb")
+  end
```

Comment:
if we need to do this locally to keep it insync do we actually need the overhead of running db:seed on startup???

I think we should choose one or the other approach. :) 

i.e. dev need to manually seed locally or reseed is done via a DB migration as we don;t need to seed on startup. :)

Reply:
yes its all the different scenario that bother me and don't know a way to achieve a good balance
- a new dev or a Dev branch runs db:prepare and this will seed anyway
- an existing dev will not run db:prepare or run.sh so relies on migrations to do it transparently or db:seed to do it locally
- a deployment would now work because of the db:seed in the run.sh but will also unnecessarily run the migration

It would be nice if to change the seeds we just add/amend the seeds and everything else just works.

Will post a question in our tech channel